libcudacxx/include/cuda/std/detail/libcxx/include/complex

```diff
@@ -1191,6 +1188,7 @@ arg(const complex<_Tp>& __c)
     return _CUDA_VSTD::atan2(__c.imag(), __c.real());
 }
 
+#ifdef _LIBCUDACXX_HAS_COMPLEX_LONG_DOUBLE
```

Comment:
@griwes This was preexisting in libcu++ 

I consider this a preexisting bug. Do you have any objections in disabling `long double` when it is not defined?